### ISSUE TITLE
Can't connect to Epic account warning on Playnite initial startup

### ISSUE THREAD

**Issue by Foofinator94** [hawkeye116477/playnite-legendary-plugin#60]
### Playnite version

10.32.0.18786

### Plugin version

1.7.1.30041

### Legendary Launcher version

0.20.34

### Bug description

When I launch playnite for the first time a day, I get a warning that the plugin fails to connect my Epic account - clicking the warning prompts the plugin to sign into Epic then goes away (I don't need to input my log in details again, it just seems the plugin isn't triggering a login upon loading playnite). Closing and Launching playnite subsequently the issue doesn't occur, only after PC restart / shutdown and reboot.
Is there a way to prevent this? 

The log file for extensions (extensions.log) snippet pasted below refers to a user path called "C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrary.cs:line 116" which I do not have on my PC. Could this be the issue?

### Relevant log

23-03 17:58:16.033|DEBUG|LegendaryLibrary#LegendaryLibrary:Found 0 installed Epic games.
23-03 17:58:17.163|ERROR|LegendaryLibrary#EpicAccountClient:Failed to validation Epic authentication.
LegendaryLibraryNS.Services.TokenException: errors.com.epicgames.common.oauth.invalid_token
   at LegendaryLibraryNS.Services.EpicAccountClient.<InvokeRequest>d__19`1.MoveNext() in C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\Services\EpicAccountClient.cs:line 271
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LegendaryLibraryNS.Services.EpicAccountClient.<GetIsUserLoggedIn>d__14.MoveNext() in C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\Services\EpicAccountClient.cs:line 165
23-03 17:58:17.177|ERROR|LegendaryLibrary#LegendaryLibrary:Failed to import linked account Epic games details.
System.Exception: User is not authenticated.
   at LegendaryLibraryNS.Services.EpicAccountClient.<GetAssets>d__15.MoveNext() in C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\Services\EpicAccountClient.cs:line 184
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at LegendaryLibraryNS.LegendaryLibrary.<GetLibraryGames>d__17.MoveNext() in C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrary.cs:line 116
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LegendaryLibraryNS.LegendaryLibrary.GetGames(LibraryGetGamesArgs args) in C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrary.cs:line 210

**Comment by hawkeye116477** [comment]
> Could this be the issue?

No, you don't need that file, it references to source code and path is from my computer :smile:. Looks like it's duplicate of #58, so test fixed version and let me know.

**Comment by Foofinator94** [comment]
Many thanks, this seems to have fixed it. I reported via the in-app reporter so wasn't aware of the repeated other issue.
